src/PokemonList/PokemonList.jsx

```diff
@@ -0,0 +1,36 @@
+import Pokemon from "../Pokemon/Pokemon";
+import './PokemonList.css';
+import PropTypes from 'prop-types';
+
+function PokemonList({ pokemons }) {
+    return (
+        <section className='pokemonList' aria-label='pokemonList'>
+            {pokemons.map((pokemon) => {
+                return (
+                    <div key={pokemon.id} aria-label='pokemon'>
+                        <Pokemon pokemon={pokemon}/>

```

Comment:
The `aria-label` that is used should match the corresponding `name` attribute on the element it is referring to. For example, `aria-label='pokemon'` is used to label the container `div` but the `Pokemon` component uses `aria-label='pokemonFigure'` rather than a name corresponding to the previously declared `aria-label`. 

Here's a good article from Google that helps to better illustrate how these attributes work together: https://developers.google.com/web/fundamentals/accessibility/semantics-aria/aria-labels-and-relationships

If you're using the `aria-label` solely for testing purposes, I would recommend using the `getByRole` and `getAllByRole` queries instead as those would be more appropriate in that use case. You can read more about the available Aria attributes and queries available here: 

- [React Testing Library - Queries](https://testing-library.com/docs/react-testing-library/cheatsheet#queries)
- [Aria Roles - MDN Docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques)